exec/InitState.hs

```diff
@@ -68,6 +71,28 @@ runBoostrap cfg action = do
         let runAction = mkRunAction (ActionEnv rp cfg logMsg)
         unNat (exceptToFail . runAction) action
 
+copyDir :: FilePath -> FilePath -> IO ()
+copyDir from to = unfoldTreeM dirTree (Left "") >>= copyTree
+  where
+    usefull = not . (`elem` [".", ".."])
```

Comment:
in fact, i am going to make this change, because now after reading and tweaking these functions for a while i get this:

```
aula-init-state: .../aula/.cabal-sandbox/share/x86_64-linux-ghc-7.10.2/aula-0.22.0/README.md: getDirectoryContents: inappropriate type (Not a directory)
```